index.html

```diff
@@ -811,6 +814,118 @@ <h3>
           </p>
         </section>
       </section>
+      <section>
+        <h3>
+          `id` member
+        </h3>
+        <p>
+          The [=manifest's=] <code><dfn data-export="" data-dfn-for=
+          "manifest" id="identity">id</dfn></code> member is a <a>string</a> that
+          represents the <dfn>identity</dfn> for the application. 
+          The [=identity=] takes the form of a URL, which is same origin as the
+          start URL.
+        </p>
+        <p>The [=identity=] is used by user agents to uniquely identify the
+        application universally. When the user agent sees a manifest with an
+        [=identity=] that does not correspond to an already-installed application,
+        it SHOULD treat that manifest as a description of a distinct
+        application, even if it is served from the same URL as that of another
+        application. When the user agent sees a manifest with an [=identity=]
+        matching that of an already-installed application, it SHOULD be used as
+        a signal that this manifest is a replacement for the already-installed
+        application's manifest, and not a distinct application, even if it is
+        served from a different URL than the one seen previously.
+        </p>
+        <p>
+        The [=identity=] is considered matching when the processed |manifest|["id"] [=url/equal|equals=].
```

Comment:
This is grammatically incorrect ("equals" what?).

Also "matching" isn't properly defined even though it's used above.

I would just delete this paragraph and then formalize the sentence in the previous paragraph:

> When the user agent sees a manifest with an identity matching that of an already-installed application, ...

becomes

> When the user agent sees a manifest _manifest_ where _manifest_["id"] [equals](https://url.spec.whatwg.org/#concept-url-equals) the identity of an already-installed application, ...

Reply:
sounds good, thanks!